### PR TITLE
Add k8s label styling

### PR DESCRIPTION
**Pull request by huwshimi**
Display the series and k8s labels in the store and entity headers with the new styling. Fixes: #3934. Fixes: #3933.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- Open /new/q/kubernetes/?type=charm
- You should see the series and k8s labels as per https://app.zeplin.io/project/5c5871a14e1897355935afb1/screen/5c9a3bc5be520e781e716534
- Click on a k8s charm and non-k8s charm, you should see the series labels in the header as per https://app.zeplin.io/project/5c5871a14e1897355935afb1/screen/5c6bc4e8d8363d9b21f7748f and https://app.zeplin.io/project/5c5871a14e1897355935afb1/screen/5c6bc4e6e66ab39ac1de43b4

File: gui/src/app/components/series-list/series-list.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* Copyright (C) 2019 Canonical Ltd. */
+'use strict';
+
+const classNames = require('classnames');
+const PropTypes = require('prop-types');
+const React = require('react');
+
+require('./_series-list.scss');
+
+class SeriesList extends React.Component {
```
Comment (hatched) [inline gui/src/app/components/series-list/series-list.js]: Nice, thanks for splitting this out!

File: gui/src/app/components/series-list/series-list.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  */
+  _generateItems() {
+    return this.props.items.map(series => {
+      const isKubernetes = series === 'kubernetes';
+      const className = classNames(
+        'p-inline-list__item',
+        'series-list__item',
+        {'series-list__item--kubernetes': isKubernetes}
+      );
+      const name = series === 'kubernetes' ? 'k8s' : series;
```
Comment (hatched) [inline gui/src/app/components/series-list/series-list.js]:
```suggestion
      const name = isKubernetes ? 'k8s' : series;
```

**Comment by hatched** [comment]
https://juju-gui-juju-pr-3935.run.demo.haus/

File: gui/src/app/components/series-list/_series-list.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+.series-list {
+  &__item {
+    display: inline-block;
+    border: 1px solid $color-mid-dark;
+    border-radius: 4px;
+    font-size: 0.875rem;
+    font-weight: 500;
+    color: $color-mid-dark;
+    padding: 0 0.5rem;
```
Comment (hatched) [inline gui/src/app/components/series-list/_series-list.scss]: This fixes the height issue for @spencerbygraves 
```suggestion
    padding: 0.1rem 0.5rem;
```

**Comment by hatched** [comment]
I unfortunately don't have permission to push to this repo so these updates will have to wait until Huw is back.

**Comment by huwshimi** [comment]
@spencerbygraves I've updated the spacing, and I check that the border and text colours are both #2e69ea (same as jaas.ai). Would you mind taking another look?

**Comment by spencerbygraves** [comment]
@huwshimi thanks, looks good now 👍

**Pull request merged**